### PR TITLE
Ai project

### PR DESCRIPTION
**Pull request by vlmoon99**

### CODE REVIEW

File: lib/constanta/storage_keys.dart
Comment (vlmoon99) [inline lib/constanta/storage_keys.dart:1]: Pls rename folder to consts

File: lib/services/ai_service.dart
Comment (vlmoon99) [inline lib/services/ai_service.dart:1]: orphographic error i nthe nme folder

File: example/lib/main.dart [hunk 1]
```diff
@@ -1,61 +1,268 @@
...
 }
 
-class MyApp extends StatefulWidget {
+class MyApp extends StatelessWidget {
   const MyApp({super.key});
 
   @override
-  State<MyApp> createState() => _MyAppState();
+  Widget build(BuildContext context) {
+    return MultiProvider(
```
Comment (vlmoon99) [inline example/lib/main.dart]: Its incorrect erchitecture of the example , pls look at this code- https://github.com/vlmoon99/flutterchain/tree/main/example  , and afetr this code take this exampels as the templete for the example - https://github.com/vlmoon99/cool_template_flutter

File: lib/services/ai_service.dart [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+import 'package:ai_model_land/services/file_interaction/local-network_service.dart';
+import 'package:ai_model_land/services/platform_info.dart';
+import 'package:ai_model_land/services/provider_ai_service.dart';
+
+class AiService {
+  final NetworkService networkInteraction;
+
+  final PlatformInfo platformInfo;
+
+  final Map<ModelFormat, ProviderAiService> provaiderService = {};
```
Comment (vlmoon99) [inline lib/services/ai_service.dart]: orthographic error , the correct name is provider

**Review by Charris1115 — COMMENTED** [review]
Command my aimodel land

**Review by Charris1115 — APPROVED** [review]

File: example/lib/main.dart (hunk 1 above)
Comment (Charris1115) [reply example/lib/main.dart]: How I'm you at this?

Comment (vlmoon99) [reply example/lib/main.dart]: > How I'm you at this?

Lol, I dont know what to say you, because Im really dont know why you participant in this review )

**Pull request merged**